pkg/build/controller/factory/factory.go

```diff
@@ -95,6 +96,10 @@ func (factory *BuildControllerFactory) Create() controller.RunnableController {
 			build := obj.(*buildapi.Build)
 			err := buildController.HandleBuild(build)
 			if err != nil {
+				// Set default Reason.
+				if build.Status.Reason == "" {
```

Comment:
Convention.  We picked one pattern and enforced it.

On Oct 11, 2015, at 3:34 PM, Rodolfo Carvalho <notifications@github.com>
wrote:

In pkg/build/controller/factory/factory.go
<https://github.com/openshift/origin/pull/4909#discussion_r41714498>:

> @@ -95,6 +96,10 @@ func (factory *BuildControllerFactory) Create() controller.RunnableController {
>  			build := obj.(*buildapi.Build)
>  			err := buildController.HandleBuild(build)
>  			if err != nil {
> +				// Set default Reason.
> +				if build.Status.Reason == "" {

I'm interested to know why do we do it that way?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4909/files#r41714498>.